zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js

```diff
@@ -1224,12 +1224,16 @@ angular.module('zeppelinWebApp')
   var setTable = function(type, data, refresh) {
     var renderTable = function() {
       var height = $scope.paragraph.config.graph.height;
-      angular.element('#p' + $scope.paragraph.id + '_table').css('height', height);
+      var container = angular.element('#p' + $scope.paragraph.id + '_table').css('height', height).get(0);
```

Comment:
i figured we could just start using the return value and save an extra lookup for the same selector. pls let me know if you have strong preference and i can revert this line.